packages/presets/src/fragments/outline/outline-viewer.ts

```diff
@@ -207,6 +219,33 @@ export class OutlineViewer extends SignalWatcher(WithDisposable(LitElement)) {
     );
   }
 
+  override firstUpdated(_changedProperties: PropertyValues) {
+    if (this._outlineViewerContainer) {
+      this._outlineViewerContainer.style.width = '0px';
+      const changeWidth = (e: TransitionEvent) => {
+        if (e.propertyName !== 'transform') return;
+        if (!this._outlineViewerContainer) return;
+
+        if (e.type === 'transitionstart' && this._showViewer) {
+          this._outlineViewerContainer.style.width = '200px';
+        }
+        if (e.type === 'transitionend' && !this._showViewer) {
+          this._outlineViewerContainer.style.width = '0px';
+        }
+      };
+      this.disposables.addFromEvent(
+        this._outlineViewerContainer,
+        'transitionstart',
+        changeWidth
+      );
+      this.disposables.addFromEvent(
+        this._outlineViewerContainer,
+        'transitionend',
+        changeWidth
+      );
```

Comment:
How about this:

```css
    .outline-viewer-root:hover .outline-heading-indicator-container {
      position: relative;
      position: flex;
      visibility: hidden;
    }

    .outline-viewer-root:hover .outline-viewer-container {
      opacity: 1;
      transform: translateX(0px);
      width: 200px;
    }
```

Reply:
If so, the animation might undergo a sudden change. By the way, the design team suggested using a different animation, which could differ from the current code. Please wait for my update.